### ISSUE TITLE
When adding Proxy Hosts, add a comment field.

### ISSUE THREAD

**Issue by wxyShine** [NginxProxyManager/nginx-proxy-manager#3213]
When adding Proxy Hosts, add a comment field.
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/42802152/7ebdd743-4bfe-4b59-9c7b-7afe4b4b59f6)

**Comment by zdcloudx** [comment]
Indeed, this feature is so important that once there are too many entries, it is difficult to distinguish the usefulness of each o

**Comment by abdulfahad661** [comment]
Yes this can be great feature enhancement for filtration of multiple doamins
